src/components/hr/MenuView/ForConversation/ForConversation.tsx

```diff
@@ -2,17 +2,19 @@ import { ButtonsAndInputView } from '../ButtonAndInputView/ButtonsAndInputView';
 import { Container, Divider } from '@mui/material';
 import theme from '../../../../theme';
 import { ReservedAccordion } from '../../../common/Accordion/ReservedAccordion';
+import { SearchValueProvider } from '../../../../context/SearchValueContext';
 
 
 export const ForConversation = () => {
 
     return (
-        <>
+        <><SearchValueProvider>
```

Comment:
Tu jak mamy jeden główny komponent to <> jest zbędne

Reply:
jasne, dzieki, poprawione :)